src/app/view/components/toolbar-add-contentlet/toolbar-add-contentlet.ts

```diff
@@ -45,11 +45,16 @@ export class ToolbarAddContenletBodyComponent {
 })
 export class ToolbarAddContenletComponent extends BaseComponent {
     @ViewChild(DropdownComponent) dropdown: DropdownComponent;
+    @Input() command?: ($event) => void;
 
-    private types: StructureTypeView[];
+    types: StructureTypeView[];
+    mainTypes: StructureTypeView[];
+    moreTypes: StructureTypeView[];
+    moreItems = [];
+    private MAIN_CONTENT_TYPES = ['CONTENT', 'WIDGET', 'FORM', 'FILEASSET', 'HTMLPAGE'];
     private recent: StructureTypeView[];
-    private structureTypeViewSelected: StructureTypeView[];
-    private showMore = false;
+    structureTypeViewSelected: StructureTypeView[];
```

Comment:
why are you removing the private?

Reply:
Private are not accessible from the template (yup weird) https://stackoverflow.com/questions/34574167/angular2-should-private-variables-be-accessible-in-the-template